### ISSUE TITLE
MQTT Map data is updated frequently after cleaning is finished

### ISSUE THREAD

**Issue by conrad784** [Hypfer/Valetudo#717]
## Describe the bug
I recently deployed monitoring to my robot (why not ;)) and noticed that my WiFi usage is pretty high (5Mb/s) when it should not be doing anything. The usage is almost zero after the nightly reboot, but after I started a cleaning cycle, the map will continuously be updated via MQTT, even when the robot is docked again.
This is not a serious issue but minor inconvenience to know that valuable data is consumed.

## To Reproduce

1. Reboot the robot
2. Start MQTT Explorer, see history of map_data --> data will be updated every ~1min.
3. Start a cleanup
4. Return to dock
5. map_data will be updated every 10s
6. after (nightly) reboot, data will update again every minute

### Vacuum Model

Roborock v1

### Valetudo Version

2021.02.0

## Expected behavior

If the robot is docked, updates of mqtt data should be less frequent (e.g. minutely)

## Additional context

Btw awesome software, I am very glad I could remove the cloud from my device :)

**Comment by Hypfer** [comment]
There is no interval in the MQTT client. Just an event listener
https://github.com/Hypfer/Valetudo/blob/f7601370353659ce8a35e6c2a942c48d4f614226/lib/mqtt/MqttClient.js#L65-L70

with the map repoll rate being determined by the robots state and reduced if it's idle
https://github.com/Hypfer/Valetudo/blob/2d02570f0c85c078941e67020e5e02a89fa62894/lib/robots/roborock/RoborockValetudoRobot.js#L270-L287

I've tested this and it works correctly on my machine.

Is there anything that might be interesting in your logs?

**Comment by conrad784** [comment]
I just started a small zoned cleanup and the result is as you described, it goes back to minutely reports.

I usually are doing full cleanups, so I will extract the debug logs next time I am doing a full cleanup. Thank you for your fast response.

**Comment by conrad784** [comment]
I am not quite sure how to debug this further.
Currently I am trying to understand the logs. Right not every minute there is a single call for `get_map_v1` and a single call to the "cloud" to `/api/miio/map_upload_handler`. In the logs above you see, that there is a call to upload every minute, but in between there are multiple `get_map_v1` calls.
If I understand correctly this function can only be executed if the last poll is `now.getTime() - 600 > this.lastMapPoll.getTime()` and what I am seeing in the logs that it is called within 3 seconds again, can only happen if `StatusStateAttribute.isActiveState`. To reach this state we will get back true for some value {RETURNING, CLEANING, MANUAL_CONTROL, MOVING} and set the repollSeconds +=2 .

Am I missing a way this could happen? Let me see if I find time this week to log those values.

**Comment by conrad784** [comment]
I just ran a full cleanup and the output is still showing this issue
Within MQTT-Explorer I can see that the data is refreshed everytime "map_upload_handler" is called. Currently this is every 1-4s. There is no "isActiveState" but the Repoll Seconds of 60 is not respected.

@Hypfer  what data can I provide to debug this further?
```
[2021-03-01T13:29:21.818Z] [INFO] started sendCloud function
[2021-03-01T13:29:21.819Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:21.820Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:21.839Z] [DEBUG] map_upload_handler { '29794537682': '' }
[2021-03-01T13:29:24.604Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4322}
[2021-03-01T13:29:24.609Z] [DEBUG] <<< cloud: {"id":4322,"result":["map_upload_handler"]}
[2021-03-01T13:29:24.610Z] [INFO] started sendCloud function
[2021-03-01T13:29:24.611Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:24.612Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:24.629Z] [DEBUG] map_upload_handler { '29794833515': '' }
[2021-03-01T13:29:25.426Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4323}
[2021-03-01T13:29:25.432Z] [DEBUG] <<< cloud: {"id":4323,"result":["map_upload_handler"]}
[2021-03-01T13:29:25.434Z] [INFO] started sendCloud function
[2021-03-01T13:29:25.435Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:25.436Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:25.454Z] [DEBUG] map_upload_handler { '29794920056': '' }
[2021-03-01T13:29:27.596Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4324}
[2021-03-01T13:29:27.601Z] [DEBUG] <<< cloud: {"id":4324,"result":["map_upload_handler"]}
[2021-03-01T13:29:27.602Z] [INFO] started sendCloud function
[2021-03-01T13:29:27.604Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:27.605Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:27.621Z] [DEBUG] map_upload_handler { '29794987140': '' }
[2021-03-01T13:29:28.140Z] [DEBUG] >>> cloud: {"method":"get_consumable","params":[],"id":4325}
[2021-03-01T13:29:28.145Z] [DEBUG] <<< cloud: {"id":4325,"result":[{"main_brush_work_time":698040,"side_brush_work_time":698040,"filter_work_time":153033,"sensor_dirty_time":22223}]}
[2021-03-01T13:29:28.381Z] [DEBUG] <<< cloud: {"method":"event.status","id":204449515,"params":[{"msg_ver":8,"msg_seq":60,"state":8,"battery":75,"clean_time":2890,"clean_area":41570000,"error_code":0,"map_present":1,"in_cleaning":0,"fan_power":75,"dnd_enabled":0}]}
[2021-03-01T13:29:28.385Z] [DEBUG] >>> cloud: {"id":204449515,"result":"ok"}
[2021-03-01T13:29:31.778Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4326}
[2021-03-01T13:29:31.790Z] [DEBUG] <<< cloud: {"id":4326,"result":["map_upload_handler"]}
[2021-03-01T13:29:31.791Z] [INFO] started sendCloud function
[2021-03-01T13:29:31.792Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:31.793Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:31.807Z] [DEBUG] map_upload_handler { '29800937682': '' }
[2021-03-01T13:29:32.857Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4327}
[2021-03-01T13:29:32.862Z] [DEBUG] <<< cloud: {"id":4327,"result":["map_upload_handler"]}
[2021-03-01T13:29:32.864Z] [INFO] started sendCloud function
[2021-03-01T13:29:32.865Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:32.866Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:32.883Z] [DEBUG] map_upload_handler { '29801091432': '' }
[2021-03-01T13:29:33.642Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4328}
[2021-03-01T13:29:33.646Z] [DEBUG] <<< cloud: {"id":4328,"result":["map_upload_handler"]}
[2021-03-01T13:29:33.651Z] [INFO] started sendCloud function
[2021-03-01T13:29:33.653Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:33.653Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:33.658Z] [DEBUG] map_upload_handler { '29801165015': '' }
[2021-03-01T13:29:37.228Z] [DEBUG] <<< cloud* {"stamp":1614605377}
[2021-03-01T13:29:37.230Z] [DEBUG] >>> cloud* {"stamp":1614605377}
[2021-03-01T13:29:37.655Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4329}
[2021-03-01T13:29:37.660Z] [DEBUG] <<< cloud: {"id":4329,"result":["map_upload_handler"]}
[2021-03-01T13:29:37.662Z] [INFO] started sendCloud function
[2021-03-01T13:29:37.663Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:37.664Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:37.684Z] [DEBUG] map_upload_handler { '29801229640': '' }
[2021-03-01T13:29:39.197Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4330}
[2021-03-01T13:29:39.202Z] [DEBUG] <<< cloud: {"id":4330,"result":["map_upload_handler"]}
[2021-03-01T13:29:39.203Z] [INFO] started sendCloud function
[2021-03-01T13:29:39.204Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:39.205Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:39.224Z] [DEBUG] map_upload_handler { '29806694306': '' }
[2021-03-01T13:29:39.488Z] [DEBUG] <<< cloud: {"id":64167289,"method":"_sync.gen_presigned_url","params":{}}
[2021-03-01T13:29:39.491Z] [DEBUG] >>> cloud: {"id":64167289,"result":["http://127.0.0.1:8079/api/miio/map_upload_handler?38170442266826","http://127.0.0.1:8079/api/miio/map_upload_handler?38170442434368","http://127.0.0.1:8079/api/miio/map_upload_handler?38170442492826","http://127.0.0.1:8079/api/miio/map_upload_handler?38170442529409"]}
[2021-03-01T13:29:44.482Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4331}
[2021-03-01T13:29:44.487Z] [DEBUG] <<< cloud: {"id":4331,"result":["map_upload_handler"]}
[2021-03-01T13:29:44.489Z] [INFO] started sendCloud function
[2021-03-01T13:29:44.490Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:44.491Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:44.510Z] [DEBUG] map_upload_handler { '29806921556': '' }
[2021-03-01T13:29:51.501Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4332}
[2021-03-01T13:29:51.506Z] [DEBUG] <<< cloud: {"id":4332,"result":["map_upload_handler"]}
[2021-03-01T13:29:51.508Z] [INFO] started sendCloud function
[2021-03-01T13:29:51.509Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:51.510Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:51.528Z] [DEBUG] map_upload_handler { '29806987640': '' }
[2021-03-01T13:29:55.024Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4333}
[2021-03-01T13:29:55.029Z] [DEBUG] <<< cloud: {"id":4333,"result":["map_upload_handler"]}
[2021-03-01T13:29:55.031Z] [INFO] started sendCloud function
[2021-03-01T13:29:55.031Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:55.032Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:55.051Z] [DEBUG] map_upload_handler { '29807033223': '' }
[2021-03-01T13:29:56.282Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4334}
[2021-03-01T13:29:56.287Z] [DEBUG] <<< cloud: {"id":4334,"result":["map_upload_handler"]}
[2021-03-01T13:29:56.289Z] [INFO] started sendCloud function
[2021-03-01T13:29:56.290Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:56.291Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:56.307Z] [DEBUG] map_upload_handler { '29812655015': '' }
[2021-03-01T13:29:57.228Z] [DEBUG] <<< cloud* {"stamp":1614605397}
[2021-03-01T13:29:57.229Z] [DEBUG] >>> cloud* {"stamp":1614605397}
[2021-03-01T13:29:58.136Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4335}
[2021-03-01T13:29:58.141Z] [DEBUG] <<< cloud: {"id":4335,"result":["map_upload_handler"]}
[2021-03-01T13:29:58.143Z] [INFO] started sendCloud function
[2021-03-01T13:29:58.144Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:58.145Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:58.161Z] [DEBUG] map_upload_handler { '29812967682': '' }
[2021-03-01T13:29:59.303Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4336}
[2021-03-01T13:29:59.308Z] [DEBUG] <<< cloud: {"id":4336,"result":["map_upload_handler"]}
[2021-03-01T13:29:59.309Z] [INFO] started sendCloud function
[2021-03-01T13:29:59.310Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:29:59.311Z] [INFO] Repoll Seconds 60
[2021-03-01T13:29:59.327Z] [DEBUG] map_upload_handler { '29813022098': '' }
[2021-03-01T13:30:02.008Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4337}
[2021-03-01T13:30:02.013Z] [DEBUG] <<< cloud: {"id":4337,"result":["map_upload_handler"]}
[2021-03-01T13:30:02.015Z] [INFO] started sendCloud function
[2021-03-01T13:30:02.016Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:02.016Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:02.033Z] [DEBUG] map_upload_handler { '29813062307': '' }
[2021-03-01T13:30:06.238Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4338}
[2021-03-01T13:30:06.244Z] [DEBUG] <<< cloud: {"id":4338,"result":["map_upload_handler"]}
[2021-03-01T13:30:06.245Z] [INFO] started sendCloud function
[2021-03-01T13:30:06.246Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:06.247Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:06.263Z] [DEBUG] map_upload_handler { '29818646723': '' }
[2021-03-01T13:30:08.580Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4339}
[2021-03-01T13:30:08.584Z] [DEBUG] <<< cloud: {"id":4339,"result":["map_upload_handler"]}
[2021-03-01T13:30:08.586Z] [INFO] started sendCloud function
[2021-03-01T13:30:08.587Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:08.588Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:08.604Z] [DEBUG] map_upload_handler { '29818741973': '' }
[2021-03-01T13:30:10.100Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4340}
[2021-03-01T13:30:10.105Z] [DEBUG] <<< cloud: {"id":4340,"result":["map_upload_handler"]}
[2021-03-01T13:30:10.106Z] [INFO] started sendCloud function
[2021-03-01T13:30:10.107Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:10.108Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:10.125Z] [DEBUG] map_upload_handler { '29818781223': '' }
[2021-03-01T13:30:14.112Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4341}
[2021-03-01T13:30:14.116Z] [DEBUG] <<< cloud: {"id":4341,"result":["map_upload_handler"]}
[2021-03-01T13:30:14.118Z] [INFO] started sendCloud function
[2021-03-01T13:30:14.119Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:14.120Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:14.139Z] [DEBUG] map_upload_handler { '29818817681': '' }
[2021-03-01T13:30:15.997Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4342}
[2021-03-01T13:30:16.002Z] [DEBUG] <<< cloud: {"id":4342,"result":["map_upload_handler"]}
[2021-03-01T13:30:16.003Z] [INFO] started sendCloud function
[2021-03-01T13:30:16.004Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:16.005Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:16.023Z] [DEBUG] map_upload_handler { '29794537682': '' }
[2021-03-01T13:30:17.228Z] [DEBUG] <<< cloud* {"stamp":1614605417}
[2021-03-01T13:30:17.230Z] [DEBUG] >>> cloud* {"stamp":1614605417}
[2021-03-01T13:30:18.588Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4343}
[2021-03-01T13:30:18.592Z] [DEBUG] <<< cloud: {"id":4343,"result":["map_upload_handler"]}
[2021-03-01T13:30:18.594Z] [INFO] started sendCloud function
[2021-03-01T13:30:18.594Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:18.595Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:18.612Z] [DEBUG] map_upload_handler { '29794833515': '' }
[2021-03-01T13:30:21.825Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4344}
[2021-03-01T13:30:21.830Z] [DEBUG] <<< cloud: {"id":4344,"result":["map_upload_handler"]}
[2021-03-01T13:30:21.831Z] [INFO] started sendCloud function
[2021-03-01T13:30:21.832Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:21.833Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:21.850Z] [DEBUG] map_upload_handler { '29794920056': '' }
[2021-03-01T13:30:24.615Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4345}
[2021-03-01T13:30:24.620Z] [DEBUG] <<< cloud: {"id":4345,"result":["map_upload_handler"]}
[2021-03-01T13:30:24.622Z] [INFO] started sendCloud function
[2021-03-01T13:30:24.623Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:24.624Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:24.640Z] [DEBUG] map_upload_handler { '29794987140': '' }
[2021-03-01T13:30:25.439Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4346}
[2021-03-01T13:30:25.445Z] [DEBUG] <<< cloud: {"id":4346,"result":["map_upload_handler"]}
[2021-03-01T13:30:25.449Z] [INFO] started sendCloud function
[2021-03-01T13:30:25.451Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:25.452Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:25.467Z] [DEBUG] map_upload_handler { '29800937682': '' }
[2021-03-01T13:30:27.607Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4347}
[2021-03-01T13:30:27.611Z] [DEBUG] <<< cloud: {"id":4347,"result":["map_upload_handler"]}
[2021-03-01T13:30:27.613Z] [INFO] started sendCloud function
[2021-03-01T13:30:27.614Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:27.614Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:27.631Z] [DEBUG] map_upload_handler { '29801091432': '' }
[2021-03-01T13:30:28.181Z] [DEBUG] >>> cloud: {"method":"get_consumable","params":[],"id":4348}
[2021-03-01T13:30:28.187Z] [DEBUG] <<< cloud: {"id":4348,"result":[{"main_brush_work_time":698040,"side_brush_work_time":698040,"filter_work_time":153033,"sensor_dirty_time":22223}]}
[2021-03-01T13:30:31.741Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4349}
[2021-03-01T13:30:31.750Z] [DEBUG] <<< cloud: {"id":4349,"result":["map_upload_handler"]}
[2021-03-01T13:30:31.753Z] [INFO] started sendCloud function
[2021-03-01T13:30:31.756Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:31.758Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:31.776Z] [DEBUG] map_upload_handler { '29801165015': '' }
[2021-03-01T13:30:32.867Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4350}
[2021-03-01T13:30:32.876Z] [DEBUG] <<< cloud: {"id":4350,"result":["map_upload_handler"]}
[2021-03-01T13:30:32.877Z] [INFO] started sendCloud function
[2021-03-01T13:30:32.878Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:32.880Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:32.895Z] [DEBUG] map_upload_handler { '29801229640': '' }
[2021-03-01T13:30:33.657Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4351}
[2021-03-01T13:30:33.662Z] [DEBUG] <<< cloud: {"id":4351,"result":["map_upload_handler"]}
[2021-03-01T13:30:33.663Z] [INFO] started sendCloud function
[2021-03-01T13:30:33.665Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:33.666Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:33.685Z] [DEBUG] map_upload_handler { '29806694306': '' }
[2021-03-01T13:30:37.229Z] [DEBUG] <<< cloud* {"stamp":1614605437}
[2021-03-01T13:30:37.230Z] [DEBUG] >>> cloud* {"stamp":1614605437}
[2021-03-01T13:30:37.666Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4352}
[2021-03-01T13:30:37.673Z] [DEBUG] <<< cloud: {"id":4352,"result":["map_upload_handler"]}
[2021-03-01T13:30:37.676Z] [INFO] started sendCloud function
[2021-03-01T13:30:37.678Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:37.680Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:37.694Z] [DEBUG] map_upload_handler { '29806921556': '' }
[2021-03-01T13:30:39.208Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4353}
[2021-03-01T13:30:39.212Z] [DEBUG] <<< cloud: {"id":4353,"result":["map_upload_handler"]}
[2021-03-01T13:30:39.214Z] [INFO] started sendCloud function
[2021-03-01T13:30:39.215Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:39.216Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:39.232Z] [DEBUG] map_upload_handler { '29806987640': '' }
[2021-03-01T13:30:39.489Z] [DEBUG] <<< cloud: {"id":76959708,"method":"_sync.gen_presigned_url","params":{}}
[2021-03-01T13:30:39.491Z] [DEBUG] >>> cloud: {"id":76959708,"result":["http://127.0.0.1:8079/api/miio/map_upload_handler?38230442771272","http://127.0.0.1:8079/api/miio/map_upload_handler?38230442945605","http://127.0.0.1:8079/api/miio/map_upload_handler?38230442991272","http://127.0.0.1:8079/api/miio/map_upload_handler?38230443025397"]}
[2021-03-01T13:30:44.493Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4354}
[2021-03-01T13:30:44.498Z] [DEBUG] <<< cloud: {"id":4354,"result":["map_upload_handler"]}
[2021-03-01T13:30:44.501Z] [INFO] started sendCloud function
[2021-03-01T13:30:44.502Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:44.502Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:44.520Z] [DEBUG] map_upload_handler { '29807033223': '' }
[2021-03-01T13:30:51.512Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4355}
[2021-03-01T13:30:51.518Z] [DEBUG] <<< cloud: {"id":4355,"result":["map_upload_handler"]}
[2021-03-01T13:30:51.519Z] [INFO] started sendCloud function
[2021-03-01T13:30:51.520Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:51.521Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:51.539Z] [DEBUG] map_upload_handler { '29812655015': '' }
[2021-03-01T13:30:55.038Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4356}
[2021-03-01T13:30:55.045Z] [DEBUG] <<< cloud: {"id":4356,"result":["map_upload_handler"]}
[2021-03-01T13:30:55.048Z] [INFO] started sendCloud function
[2021-03-01T13:30:55.050Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:55.051Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:55.066Z] [DEBUG] map_upload_handler { '29812967682': '' }
[2021-03-01T13:30:56.293Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4357}
[2021-03-01T13:30:56.298Z] [DEBUG] <<< cloud: {"id":4357,"result":["map_upload_handler"]}
[2021-03-01T13:30:56.300Z] [INFO] started sendCloud function
[2021-03-01T13:30:56.301Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:56.302Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:56.318Z] [DEBUG] map_upload_handler { '29813022098': '' }
[2021-03-01T13:30:57.229Z] [DEBUG] <<< cloud* {"stamp":1614605457}
[2021-03-01T13:30:57.230Z] [DEBUG] >>> cloud* {"stamp":1614605457}
[2021-03-01T13:30:58.148Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4358}
[2021-03-01T13:30:58.153Z] [DEBUG] <<< cloud: {"id":4358,"result":["map_upload_handler"]}
[2021-03-01T13:30:58.154Z] [INFO] started sendCloud function
[2021-03-01T13:30:58.155Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:58.156Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:58.173Z] [DEBUG] map_upload_handler { '29813062307': '' }
[2021-03-01T13:30:59.313Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4359}
[2021-03-01T13:30:59.318Z] [DEBUG] <<< cloud: {"id":4359,"result":["map_upload_handler"]}
[2021-03-01T13:30:59.319Z] [INFO] started sendCloud function
[2021-03-01T13:30:59.320Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:30:59.321Z] [INFO] Repoll Seconds 60
[2021-03-01T13:30:59.338Z] [DEBUG] map_upload_handler { '29818646723': '' }
[2021-03-01T13:31:02.019Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4360}
[2021-03-01T13:31:02.023Z] [DEBUG] <<< cloud: {"id":4360,"result":["map_upload_handler"]}
[2021-03-01T13:31:02.025Z] [INFO] started sendCloud function
[2021-03-01T13:31:02.027Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:31:02.028Z] [INFO] Repoll Seconds 60
[2021-03-01T13:31:02.043Z] [DEBUG] map_upload_handler { '29818741973': '' }
[2021-03-01T13:31:06.251Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4361}
[2021-03-01T13:31:06.256Z] [DEBUG] <<< cloud: {"id":4361,"result":["map_upload_handler"]}
[2021-03-01T13:31:06.257Z] [INFO] started sendCloud function
[2021-03-01T13:31:06.258Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:31:06.259Z] [INFO] Repoll Seconds 60
[2021-03-01T13:31:06.276Z] [DEBUG] map_upload_handler { '29818781223': '' }
[2021-03-01T13:31:08.592Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4362}
[2021-03-01T13:31:08.597Z] [DEBUG] <<< cloud: {"id":4362,"result":["map_upload_handler"]}
[2021-03-01T13:31:08.598Z] [INFO] started sendCloud function
[2021-03-01T13:31:08.600Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:31:08.601Z] [INFO] Repoll Seconds 60
[2021-03-01T13:31:08.617Z] [DEBUG] map_upload_handler { '29818817681': '' }
[2021-03-01T13:31:10.111Z] [DEBUG] >>> cloud: {"method":"get_map_v1","id":4363}
[2021-03-01T13:31:10.115Z] [DEBUG] <<< cloud: {"id":4363,"result":["map_upload_handler"]}
[2021-03-01T13:31:10.117Z] [INFO] started sendCloud function
[2021-03-01T13:31:10.118Z] [INFO] sendCloud array: ["map_upload_handler"]
[2021-03-01T13:31:10.119Z] [INFO] Repoll Seconds 60
[2021-03-01T13:31:10.134Z] [DEBUG] map_upload_handler { '29794537682': '' }
```

**Comment by Hypfer** [comment]
You can get a stacktrace with `new Error().stack`. That could be helpful to figure out where these .pollMap calls come from

**Comment by Flunkyball** [comment]
I experience the same issue. 
I did a complete factory reset and flashed a freshly built image based on v11_002034 with valetudo 2021.03.0 included using the following parameters 

`./builder_vacuum.sh     --run-custom-script=ALL \
                        --timezone=Europe/Berlin \
                        --public-key=some_key.pub \
                        --enable-greeting \
                        --disable-logs \
                        --replace-adbd \
                        --valetudo-path=../valetudo \
                        --replace-miio \
                        --enable-dns-catcher \
                        --ntpserver=172.17.5.72 \
                        --enable-addon \
                        --enable-ramdisk \
                        -f ../firmware/v11_002034.fullos.pkg`

If MQTT is activated I can see the frequent map updates while docked as well producing the log information @conrad784.
Moreover due to the "huge" network traffic the webinterface as well as the ssh connection becomes almost unresponsive. When hitting a key in the shell it takes 2-3sec for it to appear. 
After disabling MQTT everything runs directly smooth again.

**Comment by Hypfer** [comment]
This might be fixed by the MQTT rewrite in 2021.04.0

Please check and open a new issue if this is still an issue
